contracts/session/manager.go

```diff
@@ -6,5 +6,5 @@ type Manager interface {
 	// Driver retrieves the session driver by name.
 	Driver(name ...string) (Driver, error)
 	// Extend extends the session manager with a custom driver.
-	Extend(driver string, handler func() Driver) Manager
+	Extend(driver string, handler func() Driver) (Manager, error)
```

Comment:
If we are returning an error, we don't need to return the manager because chaining won't make sense to the user anyway.

Reply:
> If we are returning an error, we don't need to return the manager because chaining won't make sense to the user anyway.

makes sense.

Reply:
And I think `BuildSession` can directly pass driver name instead of handler?

Reply:
Hmm, we can pass the driver name directly but need to return an error as the second argument. Also, I believe we don't need the Driver method anymore.

cc: @hwbrzzl